addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -125,6 +125,22 @@
                                     <i class="fa fa-link mr-2"/><span t-esc="link.name"/>
                                 </span>
                             </li>
+                            <div class="o_wslides_js_course_join pl-0" t-if="slide.resource_ids">
+                                <t t-if="is_member">
+                                    <li t-foreach="slide.resource_ids" t-as="resource" class="mb-1">
+                                        <a class="o_wslides_fs_slide_link" t-attf-href="/web/content/slide.slide.resource/#{resource.id}/datas?download=true">
+                                            <i class="fa fa-download mr-2"/><span t-esc="resource.name"/>
+                                        </a>
+                                    </li>
+                                </t>
+                                <li t-elif="not is_member and slide.channel_id.enroll == 'public'" class="o_wslides_fs_slide_link mb-1">
```

Comment:
`not is_member` is not necessary
same in addons/website_slides/views/website_slides_templates_lesson.xml